### PR TITLE
DBZ-6023 [doc] Documentation for `surrogate-key` field 

### PR DESCRIPTION
**Pull request by PlugaruT**
- Document the new added field inside the `execute-snapshot` signal record in #4207

### CODE REVIEW

File: documentation/modules/ROOT/partials/modules/all-connectors/con-connector-ad-hoc-snapshots.adoc [hunk 1]
```diff
@@ -50,6 +50,12 @@ ifeval::['{context}' != 'mongodb']
 subset of the contents of the {data-collection}(s).
 endif::[]
 
+ifeval::['{context}' != 'mongodb']
+|`surrogate-key`
+|_N/A_
+| An optional string, which represents a column name that will be used during the snapshot process as a primary key of the {data-collection}(s).
```
Comment (roldanbob) [inline documentation/modules/ROOT/partials/modules/all-connectors/con-connector-ad-hoc-snapshots.adoc:56]:
```suggestion
| An optional string that specifies the column name that the connector uses as the primary key of a {data-collection} during the snapshot process.
```

File: documentation/modules/ROOT/partials/modules/all-connectors/proc-triggering-an-incremental-snapshot.adoc [hunk 2]
```diff
@@ -128,6 +128,9 @@ If you do not specify a value, the connector runs an incremental snapshot.
 | An optional string, which specifies a condition based on the column(s) of the {data-collection}(s), to capture a
 subset of the contents of the {data-collection}s.
 For more information about the `additional-condition` parameter, see xref:{context}-incremental-snapshots-additional-condition[].
+|7
+|`surrogate-key`
+|An optional string, which specifies a column name to use as the primary key for the snapshot. This allows to override the default primary key of the table.
```
Comment (roldanbob) [inline documentation/modules/ROOT/partials/modules/all-connectors/proc-triggering-an-incremental-snapshot.adoc:133]:
```suggestion
|An optional string that specifies the column name that the connector uses as the primary key for snapshots. 
The specified value overrides the default primary key of the table.
```
I think that you meant to pair this `surrogate-key` callout description with a corresponding `surrogate-key` field with the label `7` in the SQL-based snapshot example  (L63), no?

**Review by roldanbob — CHANGES_REQUESTED** [review]

File: documentation/modules/ROOT/partials/modules/all-connectors/proc-triggering-an-incremental-snapshot.adoc (hunk 2 above)
Comment (PlugaruT) [reply documentation/modules/ROOT/partials/modules/all-connectors/proc-triggering-an-incremental-snapshot.adoc:133]: My bad.

**Review by roldanbob — APPROVED** [review]
LGTM, @PlugaruT ! Thanks for your contribution!

**Comment by PlugaruT** [comment]
@roldanbob can you merge it please, I don't have the permissions

**Comment by roldanbob** [comment]
@jpechane I went ahead and merged this, since I don't think that it requires any backports.

**Comment by jpechane** [comment]
@PlugaruT @roldanbob Thanks!

**Pull request merged**